restaurant/src/app/admin/update-status.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient,HttpErrorResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UpdateStatusService {
+ 
+  
+  
+  constructor(private http:HttpClient) { }
+
+  update(){
+     var id = localStorage.getItem('update');
```

Comment:
we can pass id to this method. no need for `var id = localStorage.getItem('update');`